.gitignore

```diff
@@ -0,0 +1 @@
+build/
```

Comment:
I think we should exclude this from this PR. First, the PR is said to update the readme, which makes this is outside of this scope. Second, folks can have their build directory named however they want and stored inside or outside the build tree, so I don't think it makes sense to add a special case for `build`.